src/algebra/star/chsh.lean

```diff
@@ -0,0 +1,265 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison.
+-/
+import algebra.star.algebra
+import algebra.algebra.ordered
+import analysis.special_functions.pow
+
+/-!
+# The Clauser-Horne-Shimony-Holt inequality and Tsirelson's inequality.
+
+We establish a version of the Clauser-Horne-Shimony-Holt (CHSH) inequality
+(which is a generalization of Bell's inequality).
+This is a foundational result which implies that
+quantum mechanics is not a local hidden variable theory.
+
+As usually stated the CHSH inequality requires substantial language from physics and probability,
+but it is possible to give a statement that is purely about ordered *-algebras.
```

Comment:
Can you put backticks around all the `*` in "`*-algebras`"? It looks like currently they're getting confused for emphasis marks: https://observablehq.com/@bryangingechen/github-lean-doc-preview?url=%224687%22#docs
```suggestion
but it is possible to give a statement that is purely about ordered `*`-algebras.
```

Reply:
Done.